ecs-init/cache/cache_test.go

```diff
@@ -191,13 +201,16 @@ func TestDownloadAgentDownloadTarballNon200(t *testing.T) {
 	mockFS := NewMockfileSystem(mockCtrl)
 	mockFS.EXPECT().MkdirAll(config.CacheDirectory(), os.ModeDir|0700)
 	mockGetter := NewMockhttpGetter(mockCtrl)
-	mockGetter.EXPECT().Get(config.AgentRemoteTarballMD5()).Return(md5response, nil)
+	mockGetter.EXPECT().Get(config.AgentRemoteTarballMD5(config.DefaultRegionName)).Return(md5response, nil)
```

Comment:
Same as above. Can we be more strict and the check the ordering here (Using [`gomock.InOrder`](https://godoc.org/github.com/golang/mock/gomock#InOrder))?